app/controllers/review_mapping_controller.rb

```diff
@@ -324,7 +257,8 @@ def list
 
     @letters = []
     @assignments = Assignment
-                   .where(["instructor_id = ? and substring(name,1,1) = ?", session[:user].id, letter])
+                   .where(instructor_id:session[:user].id)
```

Comment:
I enter rails console, and use some data instead of parameters to test the result after running the previous version and the latest version of query like:
ReviewResponseMap.where(['reviewee_id = ? and reviewer_id = ? ‘,6050, 9])
ReviewResponseMap.where(reviewee_id: 6050,reviewer_id: 9)